authors/handlers.go

```diff
@@ -89,30 +89,46 @@ func (h *AuthorHandler) GetCount(writer http.ResponseWriter, req *http.Request)
 }
 
 // HealthCheck - Return FT standard healthcheck
-func (h *AuthorHandler) HealthCheck() v1a.Check {
-
-	return v1a.Check{
+func (h *AuthorHandler) HealthCheck() fthealth.Check {
+	return fthealth.Check{
 		BusinessImpact:   "Unable to respond to requests",
-		Name:             "Check service has finished initilising.",
-		PanicGuide:       "https://sites.google.com/a/ft.com/ft-technology-service-transition/home/run-book-library/v1-authors-transformer",
+		Name:             "Check service has finished initialising.",
+		PanicGuide:       "TBD",
 		Severity:         1,
 		TechnicalSummary: "Cannot serve any content as data not loaded.",
 		Checker: func() (string, error) {
```

Comment:
Shouldn't we call the method `initChecker` here?

Reply:
Well the behavior of `__gtg` and `__health` wasn't the same before, maybe for a reason?